<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>859</width>
    <height>140</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="startButton">
        <property name="text">
         <string>Start/Stop</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Mindwave device</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="deviceComboBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <item>
         <property name="text">
          <string>/dev/rfcomm0</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>/dev/rfcomm1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>/dev/rfcomm2</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>/dev/ttyUSB0</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>/dev/ttyUSB1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>/dev/ttyUSB2</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>/dev/ttyS0</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>/dev/ttyS1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>/dev/ttyS2</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>/dev/tty0</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>/dev/tty1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>/dev/tty2</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Eye blink sensitivity</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="eyeBlinkSensitivity">
        <property name="text">
         <string>0.45</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Lower Eye blink frequency  (Hz)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lowerEyeBlinkFrequency">
        <property name="text">
         <string>5.0</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Higher Eye blink frequency (Hz)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="higherEyeBlinkFrequency">
        <property name="text">
         <string>20.0</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>84</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>859</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionSave_state">
   <property name="text">
    <string>Save state</string>
   </property>
  </action>
  <action name="actionLoad_state">
   <property name="text">
    <string>Load state</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>deviceComboBox</tabstop>
  <tabstop>eyeBlinkSensitivity</tabstop>
  <tabstop>lowerEyeBlinkFrequency</tabstop>
  <tabstop>higherEyeBlinkFrequency</tabstop>
  <tabstop>startButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
